Woodoku/src/Entity/WoodokuBoard.py

```diff
@@ -22,86 +25,167 @@ def remove_blocks(self, blocks_coord: List[Tuple[int, int]]) -> None:
         pass
 
     def is_occupied(self, blocks_coord: List[Tuple[int, int]]) -> bool:
-        """
-        Check if each coord has block on it. If all of those coords are occupied, return true. Otherwise, false.
+        """Check if each coord has a block on it. If all of those coords are
+        occupied, return true. Otherwise, false.
+
+        Args:
+            blocks_coord (List[Tuple[int, int]]): list of coordinates to check
+
+        Raises:
+            CannotPlaceShapeError: raised when checking a invalid block
 
-        :param blocks_coord:
-        :return:
+        Returns:
+            bool: if any coordinates in `blocks_coord` is occupied
         """
-        pass
+        # TODO: raise the Error, according to docstring, the code below is just
+        # a placeholder
```

Comment:
Might want to delete this comment. Instead, create a new issue with detailed explanation.